app/services/idv/steps/inherited_proofing/agreement_step.rb

```diff
@@ -14,6 +16,20 @@ def form_submit
         def consent_form_params
           params.require(:inherited_proofing).permit(:ial2_consent_given)
         end
+
+        def enqueue_job
```

Comment:
We may have to deal with the case of a double submit here. Meaning if there is another job in progress we don't want to enqueue another one.

Reply:
Yeah, something like that. Checking to see if there is a uuid in the session already. If there is looking up the job and making sure it is not in-progress

Reply:
Will do. Thanks for the feedback